src/transitionEnd.ts

```diff
@@ -49,14 +50,15 @@ function transitionEnd(
   handler: Listener,
   duration?: number
 ) {
-  if (!TRANSITION_SUPPORTED) {
-    return emulateTransitionEnd(element, 0, 0)
-  }
```

Comment:
i think we just don't need this? you can still read the transition duration on IE11...